### DISCUSSION TITLE
Changing the playing field of osu!mania on android/ios version

### DISCUSSION THREAD

**Discussion by KozacheShapka** [ppy/osu#31179] [Ideas] [answered]
I’d like to propose a change to the Mania mode on the Android/ios version of osu! to improve the gameplay experience.

Currently, the large on-screen buttons covering almost half the screen can feel a bit cumbersome, especially on smaller devices. I suggest making the playfield horizontal, so the notes are more visible and the gameplay becomes more dynamic and comfortable.

Additionally, I propose reverting to the old system where invisible touch zones are placed directly under the judgment line, instead of using large buttons for tapping. This would allow players to tap in rhythm without obstructing the view of the notes, similar to how it worked before.

I’ve attached an image to better illustrate how I envision this change. It shows the horizontal playfield.
And, I think this adjustment would improve the overall gameplay experience for Mania on mobile.

_(I use machine translation.)_
![osu!mania_android](https://github.com/user-attachments/assets/1e707fc2-85df-4c44-b99f-cd246fee9594)

**Comment by Joehuu** [comment, marked as answer]
This is planned, but is on the back burner. https://github.com/ppy/osu-framework/pull/6062 needs to happen first.
